docs/CW.md

```diff
@@ -197,15 +189,15 @@ Arok is in the process of adding this field to the database. [2021/03/16] This f
 
 It's unclear how this field is to be interpreted when there are multiple morphemes in `\mrp`. A good default assumption is that `\mrp2` only shows the morpheme breakdown for the stem in `\mrp` (so `\mrp2` is essentially the stem components—initial, medial, and final).
 
-## `\new` **neologism** [multiple]
+## `\new` **new since 2001 publication** [multiple]
 
-The value of this field is always `new` (with perhaps one exception; see below). This field seems to function as a Boolean, indicating (presumably) that the entry is a neologism. It might also be a field Arok uses for himself, to keep track of which entries are new.
+The value of this field is always `new` (with perhaps one exception; see below). This field indicates that the entry was added after the publication of the 2001 edition of the _Cree: Words_ dictionary was published.
 
-In at least one case, this field appears multiple times, with one of the fields blank, and the other containing `new`.
+Not all entries since 2001 are marked as new. These could potentially be determined based on information in the `\src` field. In at least one case, this field appears multiple times, with one of the fields blank, and the other containing `new`.
 
 ## `\ps` **part of speech** [multiple]
 
-This field really combines information on part of speech, morpheme type, and inflectional class.
+This field really combines information on part of speech, morpheme type, and inflectional class. If an entry has multiple `\ps` fields, those should each be given their own sense (with the same definition, if necessary).
```

Comment:
Note that for Indeclinable Particles, the "inflectional class" is rather a subtype of a particle than a real inflectional class.